app/controllers/cookies_consent_controller.rb

```diff
@@ -0,0 +1,35 @@
+require 'json'
+
+class CookiesConsentController < ApplicationController
+  def self.get_cookie_policy_value(cookie)
```

Comment:
```suggestion
  helper_method :get_cookie_policy_value(cookie)

  def get_cookie_policy_value(cookie)
```

Reply:
I think it makes sense to have this as a protected helper and possibly to do a rename. 

You can add a section for protected methods above the private ones:

```ruby
protected
  def policy_cookie
    cookies['cookies_policy'].present? JSON.parse(cookies['cookies_policy']') : {}
  end

```

And consume it with:

```ruby
policy_cookie['usage']
```

Reply:
We basically don't usually label getters/setters with get_/set_ in the land of ruby

Reply:
And:

```ruby
  helper_method :policy_cookie
```